qt/scientific_interfaces/ISISReflectometry/Common/ModelCreationHelper.h

```diff
@@ -0,0 +1,93 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+#ifndef MANTID_CUSTOMINTERFACES_MODELCREATIONHELPER_H_
+#define MANTID_CUSTOMINTERFACES_MODELCREATIONHELPER_H_
+
+#include "../../ISISReflectometry/Reduction/Experiment.h"
+#include "../../ISISReflectometry/Reduction/Instrument.h"
+#include "../../ISISReflectometry/Reduction/ReductionJobs.h"
+#include "Common/DllConfig.h"
+
+namespace MantidQt {
+namespace CustomInterfaces {
+namespace ModelCreationHelper {
+
+/* Helper methods to create reduction configuration models for the
+ * reflectometry GUI tests */
```

Comment:
Ok this explains fake values. Would be useful if this was in some kind of test namespace or directory.